src/script-to-express.ts

```diff
@@ -21,6 +22,9 @@ export function scriptToExpress(source: string) {
       const result = await runner(bodyToEvent(req.body));
       if(result === undefined) {
         res.status(400)
+        if(req.body.resolver === '$webhook') {
+          res.status(200)
+        }
       }
```

Comment:
```suggestion
      if(result === undefined && req.body.resolver !== '$webhook') {
        res.status(400)
      }
```

does that work same?

Reply:
Yes